index.js

```diff
@@ -12,10 +12,26 @@ let deploymentConfig
 module.exports = (robot, { getRouter }, Settings = require('./lib/settings')) => {
   async function syncAllSettings (nop, context, repo = context.repo(), ref) {
     try {
-      deploymentConfig = await loadYamlFileSystem()
-      robot.log.debug(`deploymentConfig is ${JSON.stringify(deploymentConfig)}`)
+      let deploymentConfig;
+      try {
+        deploymentConfig = await loadYamlFileSystem();
+      } catch (error) {
+        // Assumption here is if some one is using just repo config they don't need include exclude list of repos
```

Comment:
Wouldn't it be better to create a feature flag, instead of assuming this functionality? Assuming it can create confusion and not leave the execution logic clear. What is your opinion ?

Reply:
ya agree